fastlane/Fastfile

```diff
@@ -277,17 +321,47 @@ platform :ios do
   #####################################################################################
   desc 'Updates a release branch for a new beta release'
   lane :new_beta_release do |options|
-    ios_betabuild_prechecks(options)
+    # Checkout default branch and update
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    # Check versions
+    message = "The current build code has been detected: #{current_build_code}\n"
+
+    # Check branch
+    UI.user_error!("#{message}Release branch for version #{current_release_version} doesn't exist. Abort.") unless !options[:base_version].nil? || Fastlane::Helper::GitHelper.checkout_and_pull(release: current_release_version)
+
+    # Check user overwrite
+    switch_to_user_branch(options[:base_version], message) unless options[:base_version].nil?
+
+    # Verify
+    message << "Updating branch to version: #{next_build_code}.\n"
+    if options[:skip_confirm]
+      UI.message(message)
+    else
+      UI.user_error!('Aborted by user request') unless UI.confirm("#{message}Do you want to continue?")
+    end
+
+    # Check local repo status
+    ensure_git_status_clean
+
+    # Download the latest translations from GlotPress
     download_localized_strings_and_metadata_from_glotpress
     lint_localizations
-    ios_bump_version_beta
+
+    # Bump the build code
+    UI.message 'Bumping build code...'
+    Fastlane::Helper::GitHelper.ensure_on_branch!('release')
```

Comment:
Good to know! I'm pretty sure I spotted some uses of the built-in action. I'm going to hold off on changing all of these over to the built-in action for a separate PR. I've tried to just replicate everything that was in the precheck and version bump actions so that we reduce the possibility of introducing unexpected behavior with the rest of these changes.

Reply:
Addressed here: https://github.com/woocommerce/woocommerce-ios/pull/10807/commits/b8802015356361954a34b2f3ea6e0241483966a7